python/lsst/meas/astrom/matchPessimisticB.py

```diff
@@ -505,7 +503,6 @@ def _doMatch(self, refCat, sourceCat, wcs, refFluxField, numUsableSources,
         dist_cut_arcsec = np.max(
             (2 * wcs.pixelScale().asArcseconds(),
              clipped_dist_arcsec, maxMatchDistArcSec))
-        print("Dist cut", dist_cut_arcsec, "arcsec")
```

Comment:
These where more for me to check what was going on in the unittest when I was trying to update the code. The useful information about matching is already stored in the debug logger.